### ISSUE TITLE
Some minor issues with index entries

### ISSUE THREAD

**Issue by kentemorrison** [OpenDiscreteMath/ibl-combinatorics#109]
The latex/pdf version seems to be in good enough shape to use in your classes, so I'm not sure that we should worry now about these little things (and any others). I noticed these since the last pull requests that I thought wrapped up the index editing.

1) Accented characters throw off the alphabetization in the index. There is ménage and Pólya. The first I changed to menage but the second I didn't catch. It's at the end of the P's. 

2) There is an entry for "range" in Bogart's pdf and not in our update. But the text where range is defined is different. In the update "range" is not defined. This is in section A.1.1.

**Comment by oscarlevin** [comment]
2 should be fixed.  I updated that in my index fix (and added the text about domain and range).  Did you pull the latest update-bogart from upstream?

**Comment by kentemorrison** [comment]
Thought I had but evidently not. Now I have pulled the latest and see the fix. 

Now a further issue. This definition of range is not the image of the function but rather what is often called the target or codomain. However, changing from range to codomain, say, would require changing other parts of the text. I recommend leaving it alone but being aware of the possibility of confusion with the usage of "range" from calculus.

**Comment by oscarlevin** [comment]
Yeah, I had to grit my teeth when writing that sentence.  In the effort to create a unabridged copy of the text, I thought I would leave it for now, but we should remember to update it when an updated edition is considered.

**Comment by mitchkeller** [comment]
I'm always careful to warn my intro to proofs students that what the text I use calls range and codomain other books might call image and range. It's annoying that we can't agree 100% as a community on this. I'm inclined to support making an edit on that once we declare "Here's the faithful Bogart." and can think about tweaking it.

**Comment by mitchkeller** [comment]
I'm thinking that the issue about accented characters and index alphabetization should be considered an upstream issue with PreTeXt and punted up for Rob to resolve. We can do a quick hand edit on the PDF before we send it to CreateSpace for this go around, but then this could be closed and punted. Anyone care to disagree?

**Comment by oscarlevin** [comment]
Sounds good to me.

**Comment by mitchkeller** [comment]
I figured out how to deal with the accented characters by lightly modifying the LaTeX. I'll file an issue on PreTeXt noting what fix I used.

We should keep this issue open in part because there's the matter of "range" that we discussed in August and also in taking a peek at the index, I noticed some weird inconsistencies with the two types of partitions the book talks about in terms of indexing. But if we're going for faithful Bogart at this stage, I'm not going to fix them now. (There's also "pair,ordered" in the index that looks like it needs to have a subindex entry instead.)

**Comment by mitchkeller** [comment]
In looking at the PreTeXt schema, I think the fix we need is already built-in by using the @sortby attribute on idx. Because I don't want to make a mess out of things, I'm going to wait for #125 to get merged before I do any edits, but I think I can fix the PTX code to deal with the sorting permanently. That will still leave a number of things in my previous comment to address, however.
